packages/js/plugins/uri-resolvers/file-system-resolver/src/index.ts

```diff
@@ -37,7 +36,7 @@ export class FileSystemResolverPlugin extends Module<NoConfig> {
     if (manifestExistsResult.data) {
       try {
         const manifestResult = await FileSystem_Module.readFile(
-          { path: manifestPath, encoding: FileSystem_EncodingEnum.BINARY },
```

Comment:
Why were these changes made? I'm going to revert them as they're causing tests to fail.

Reply:
My mistake! I just realized that the `encoding` property was removed from the `readFile` method's arguments, and wasn't causing an error because the arguments used to be a generic object. With the new release this is causing an error because the property does not exist. My bad @krisbitney